pystiche_papers/gatys_ecker_bethge_2015/_loss.py

```diff
@@ -25,7 +25,9 @@ def __init__(
     ):
         super().__init__(encoder, score_weight=score_weight)
 
+        # https://github.com/pmeier/PytorchNeuralStyleTransfer/blob/master/NeuralStyleTransfer.ipynb [3]
```

Comment:
1. Cell `[3]` defines the style loss, but we need the content loss here.
2. Please add a little more explanation why this cell shows that we need a `score_correction_factor`. Here that would be something like: "torch.nn.MSELoss() was used to calculate the content loss, which does not include the factor 1/2 given in the paper".